integration_test/cloudbuild-integration-test.yaml

```diff
@@ -42,7 +42,9 @@
   id: adapter-main
   entrypoint: 'bash'
   args: [ '-c',
-          "/import/bin/import \
+          "java -Dorg.dcm4che3.imageio.codec.ImageReaderFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageReaderFactory.properties \
+                -Dorg.dcm4che3.imageio.codec.ImageWriterFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageWriterFactory.properties \
+                -cp /app/resources:/app/classes:/app/libs/* com.google.cloud.healthcare.imaging.dicomadapter.ImportAdapter \
```

Comment:
Just wondering, why does the -cp (class path) need to be specified to this directory? could you add a brief comment explaining why?

Reply:
This is exactly same command as image's entrypoint. Relevant part of dockerfile: 
```
WORKDIR /app 
COPY dir:84ad222acc46d4d7aac43bdbfba24beea33f6c179076d1e3435ad95395bf7930 in libs/
COPY dir:dc25dd37b9b376d9e5fe63d6c93793a16e04ac13ac9d89928134c6d2128447ae in resources/
COPY dir:e0862f2c38b8b35af27e97667325f199037ee6a734e2c2d480795024f5333dd7 in classes/
ENTRYPOINT ["java" "-Dorg.dcm4che3.imageio.codec.ImageReaderFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageReaderFactory.properties" "-Dorg.dcm4che3.imageio.codec.ImageWriterFactory=com/google/cloud/healthcare/imaging/dicomadapter/transcoder/ImageWriterFactory.properties" "-cp" "/app/resources:/app/classes:/app/libs/*" "com.google.cloud.healthcare.imaging.dicomadapter.ImportAdapter"] 
```